profitbricks/src/main/java/org/jclouds/profitbricks/domain/LoadBalancer.java

```diff
@@ -0,0 +1,398 @@
+         this.loadBalancerAlgorithm = loadBalancerAlgorithm;
+         return this;
+      }
+
+      public Builder dataCenterId(String dataCenterId) {
+         this.dataCenterId = dataCenterId;
+         return this;
+      }
+
+      public Builder dataCenterVersion(String dataCenterVersion) {
+         this.dataCenterVersion = dataCenterVersion;
+         return this;
+      }
+
+      public Builder internetAccess(boolean internetAccess) {
+         this.internetAccess = internetAccess;
+         return this;
+      }
+
+      public Builder ip(String ip) {
+         this.ip = ip;
+         return this;
+      }
+
+      public Builder lanId(String lanId) {
+         this.lanId = lanId;
+         return this;
+      }
+
+      public Builder creationTime(Date creationTime) {
+         this.creationTime = creationTime;
+         return this;
+      }
+
+      public Builder state(ProvisioningState state) {
+         this.state = state;
+         return this;
+      }
+
+      public Builder lastModificationTime(Date lastModificationTime) {
+         this.lastModificationTime = lastModificationTime;
+         return this;
+      }
+
+      public Builder balancedServers(List<Server> balancedServers) {
+         this.balancedServers = balancedServers;
+         return this;
+      }
+
+      public Builder firewalls(List<Firewall> firewalls) {
+         this.firewalls = firewalls;
+         return this;
+      }
+
+      public LoadBalancer build() {
+         return LoadBalancer.create(id, name, loadBalancerAlgorithm, dataCenterId, dataCenterVersion, internetAccess, ip, lanId, state, creationTime, lastModificationTime, balancedServers, firewalls);
+      }
+
+      public Builder fromLoadBalancer(LoadBalancer in) {
+         return this.id(in.id()).name(in.name()).loadBalancerAlgorithm(in.loadBalancerAlgorithm())
+                 .dataCenterId(in.dataCenterId()).dataCenterVersion(in.dataCenterVersion()).internetAccess(in.internetAccess())
+                 .ip(in.ip()).lanId(in.lanId()).state(in.state()).creationTime(in.creationTime()).lastModificationTime(in.lastModificationTime()).balancedServers(in.balancedServers()).firewalls(in.firewalls());
+      }
+   }
+
+   public static final class Request {
+
+      public static CreatePayload.Builder creatingBuilder() {
+         return new CreatePayload.Builder();
+      }
+
+      public static UpdatePayload.Builder updatingBuilder() {
+         return new UpdatePayload.Builder();
+      }
+
+      public static RegisterPayload.Builder registerBuilder() {
+         return new RegisterPayload.Builder();
+      }
+
+      public static DeregisterPayload.Builder deregisterBuilder() {
+         return new DeregisterPayload.Builder();
+      }
+
+      @AutoValue
+      public abstract static class CreatePayload {
+
+         public abstract String dataCenterId();
+
+         public abstract String loadBalancerName();
+
+         public abstract LoadbalancerAlgorithm loadBalancerAlgorithm();
```

Comment:
Add nullable as well, since it's not required according to [docs](https://www.profitbricks.com/apidoc/1_3/APIDocumentation.html?CreateLoadBalancer.html)